meeting-search.php

```diff
@@ -102,6 +95,7 @@
         <?php } ?>
 
     <Say voice="<?php echo setting('voice') ?>" language="<?php echo setting('language') ?>">
+	<?php echo word('meeting_search_more_info')?>
```

Comment:
It’s already possible to override the goodbye message without making this change.

Reply:
The only place I could find to override the goodbye message was in the en-US.php (or other appropriate language file).  I was just trying to make it language agnostic and minimize editing files other than config.php on reinstall or upgrade.  Not a big issue just thought it was cleaner on the multi-language.

Reply:
I missed that.